css/theme/clermontech.css

```diff
@@ -5,14 +5,14 @@
  */
 @import url(//fonts.googleapis.com/css?family=Maven+Pro);
 @font-face {
-  font-family: Neutraface;
-  src: url("../../lib/font/neutraface-condensed-light.otf");
+  font-family: Nanum;
```

Comment:
Quoting wikipedia (emphasis mine):

> The Nanum fonts [...] is a series of open source unicode fonts **designed for the Korean language**

As you can see, each file is almost 4MB.  Is this intentional to have more than 10MB of resources to download in order to render a single static page website?

Unless we need some extended unicode support, we can probably stick with a simple, lighter font.  Don't know what your acceptance criteria are, found some matches by searching for "thin sans-serif OFL font".

Reply:
Fair point. I'm looking for a font to replace Neutraface with something similar.
But it's not easy to find something good enough...
IMO having a $MB font is not an issue for this repo since we use it only locally to introduce the ApiHour.

But I used it also for the planet, where we serve the font via a github page...
https://github.com/clermontech/planet/pull/289